_pages/integrations/azure_listeners.md

```diff
@@ -50,8 +50,8 @@ Create a new guest user account with Algorithmia’s account email.
 ## 4. Create Service Bus Namespace and Queue 
 
 * Import ARM template and create your Service Bus Namespace and Queue.
-Info about ARM templates: https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/overview
-Info about Service Bus: https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#:~:text=In%20the%20left%20navigation%20pane,if%20the%20name%20is%20available
+<a href="https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/overview" target="_blank">Info about ARM templates</a>
+<a href="https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#:~:text=In%20the%20left%20navigation%20pane,if%20the%20name%20is%20available" target="_blank">Info about Service Bus</a>
```

Comment:
For all the cases on this page where we're using `target="_blank"` to external sites, can you add a `rel="noopener noreferrer"` attribute? It prevents some common security issues.

Reply:
sure thing sounds good